lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -111,10 +110,22 @@ const getMeasureLibraryNameAndCql = (matMeasure: MatMeasure): { cqlLibraryName:
   const cqlBuffer = Buffer.from(cqlContent.data, "base64");
   // convert base64 string to text cql
   const textCql = cqlBuffer.toString("ascii");
+  const matchedCqlWithRegExpression = textCql.match(/^using FHIR.*/gm);
+  let textCqlAfterReplacing = "";
+  if (matchedCqlWithRegExpression) {
+    let replacedTextCql = textCql;
+    textCqlAfterReplacing = matchedCqlWithRegExpression.reduce((matchedCql) => {
+      replacedTextCql = replacedTextCql.replace(
+        matchedCql,
+        matchedCql.replace(/^using FHIR.*/gm, "using QICore version '4.1.0'"),
+      );
+      return replacedTextCql;
+    }, replacedTextCql);
+  }
```

Comment:
Considering using FHIR version '4.0.1' is going to be constant, would this work?
`textCql.replace("using FHIR version '4.0.1'", "using QICore version '4.1.0'")`

Reply:
Yeah it does work, but I wasn't sure if the version value is constant(in our case it is 4.0.1) so I have used regular expression to match it and then replace it